package.json

```diff
@@ -20,10 +20,9 @@
     "build": "tsc && pnpm run build:templates",
     "build:templates": "cp -r src/server/templates dist/src/server/",
     "format": "biome check --write",
-    "test": "pnpm run test:unit && pnpm run test:type && pnpm run test:format",
```

Comment:
please keep this in order to be able to run locally and iterate on a specific dev